subsys/bluetooth/mesh/shell/Kconfig

```diff
@@ -0,0 +1,13 @@
+# Bluetooth mesh shell configuration options
+
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_MESH_SHELL_HEALTH_SRV
```

Comment:
```suggestion
config BT_MESH_SHELL_ADD_HEALTH_SRV_INSTANCE
```

Reply:
How about `BT_MESH_SHELL_HEALTH_SRV_INSTANCE`? I think the `ADD` is implicit in this case, and if the user is wodering they could/should always read the help box.